dds/DCPS/MulticastManager.cpp

```diff
@@ -41,25 +40,23 @@ bool MulticastManager::process(InternalDataReader<NetworkInterfaceAddress>::Samp
 
     switch (info.kind) {
     case ISIK_REGISTER:
-      if (log_level >= LogLevel::Notice) {
-        ACE_ERROR((LM_NOTICE, "(%P|%t) NOTICE: MulticastManager::process: unexpected REGISTER\n"));
-      }
+      // Ignore.
       break;
     case ISIK_SAMPLE: {
       if (nia.exclude_from_multicast(multicast_interface.c_str())) {
         continue;
       }
 
       if (joined_interfaces_.count(nia.name) == 0 && nia.is_ipv4()) {
-        if (log_level >= LogLevel::Info) {
-          ACE_DEBUG((LM_INFO,
-                     "(%P|%t) MulticastManager::process: joining group %C on %C\n",
-                     LogAddr(multicast_group_address).c_str(),
-                     nia.name.empty() ? "all interfaces" : nia.name.c_str()));
-        }
-
         if (0 == multicast_socket.join(multicast_group_address.to_addr(), 1, nia.name.empty() ? 0 : ACE_TEXT_CHAR_TO_TCHAR(nia.name.c_str()))) {
           joined_interfaces_.insert(nia.name);
+          if (log_level >= LogLevel::Info) {
+            ACE_DEBUG((LM_INFO,
+                       "(%P|%t) INFO: MulticastManager::process: joined group %C on %C (joined count %B)\n",
```

Comment:
```suggestion
            ACE_DEBUG((LM_INFO,
                       "(%P|%t) MulticastManager::process: joined group %C on %C (joined count %B)\n",
```
Same thing in some other changes where LM_INFO is used.

Reply:
I'm going to update the logging guidelines because as a user, I would like consistency knowing that these are info messages.